src/nopoll_io.c

```diff
@@ -66,7 +66,7 @@ noPollPtr nopoll_io_wait_select_create (noPollCtx * ctx)
  * 
  * @param fd_group The fd group to be deallocated.
  */
-void    nopoll_io_wait_select_destroy (noPollCtx * ctx, noPollPtr fd_group)
+void    nopoll_io_wait_select_destroy (noPollCtx * ctx, noPollPtr __fd_group)
```

Comment:
variables with `double underscores` are generally reserved by libc as private namespace.

Reply:
Numerous nopoll modules violate this rule, with numerous function names, parameter names, and variable names that start with double underscores.
So the question for us would be how to deal with software we didn't write that has already been written to a standard we don't agree with.
In this specific case, I thought it was best to just conform to the naming convention already being used in this module, nopoll_io.c.